src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -603,22 +636,33 @@ def _get_vm_set_type(self, read_only: bool = False, **kwargs) -> Union[str, None
             return vm_set_type
 
         # dynamic completion
-        # the value verified by the validator may have case problems, and function "_set_vm_set_type" will adjust it
+        # the value verified by the validator may have case problems, and we will adjust it by default
         if not read_from_mc:
-            vm_set_type = _set_vm_set_type(
-                vm_set_type=vm_set_type,
-                kubernetes_version=self.get_kubernetes_version(),
-            )
+            kubernetes_version = self.get_kubernetes_version()
+            if not vm_set_type:
+                if kubernetes_version and StrictVersion(kubernetes_version) < StrictVersion("1.12.9"):
+                    print("Setting vm_set_type to availabilityset as it is \
```

Comment:
what's this print for?

Reply:
As a hint to user? This is the original behavior defined in `_set_vm_set_type`.